src/notifications/background/index.js

```diff
@@ -0,0 +1,70 @@
+import { makeRemotelyCallable } from 'src/util/webextensionRPC'
+import NotificationStorage from './storage'
+import * as notifications from '../notifications'
+
+export default class NotificationBackground {
+    static LAST_NOTIF_TIME = 'last-notif-releast-time'
```

Comment:
Looks like a typo. Maybe it should be something like `last-notif-proc-timestamp` to better indicate its purpose? (a bit subjective maybe but possibly they could be processed independently of releases, and "timestamp" is more in-line with recent storage keys holding timestamp values)